internal/provider/harness_container_resource.go

```diff
@@ -255,3 +214,61 @@ func (r *HarnessContainerResource) Delete(ctx context.Context, req resource.Dele
 func (r *HarnessContainerResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
 }
+
+// addContainerResourceSchemaAttributes adds common container resource
+// attributes to the given map. this function is provided knowing how common it
+// is for other harnesses to require some sort of container configuration.
+func addContainerResourceSchemaAttributes(attrs map[string]schema.Attribute) map[string]schema.Attribute {
```

Comment:
Does this need to take an argument? It's a private function with just one usage that passes an empty map in.

Reply:
as it sits today no, but I left it in just in case a caller has a non nil use case 🤷

Reply:
May I suggest
```suggestion
func addContainerResourceSchemaAttributes(_ map[string]schema.Attribute) map[string]schema.Attribute {
```
to make it clear it's unused right now?

Reply:
I just removed the field from the signature for now as per the first commend. will add it back later when its needed!